counting_elements.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Uniter PHP Counting Current Elements Demo</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <script src="/assets/jquery-full.js"></script>
+        <link rel="stylesheet" href="css/bootstrap.min.css">
+        <link rel="stylesheet" href="css/style.css">
+    </head>
+    <body id="counting_elements">
+
+        <div class="full_row">
+            <h1>
+                Uniter PHP Counting Current Elements Demo
+            </h1>
+            <p>
+                The logic for this page is written entirely in PHP,
+                see <a href="//github.com/PharaohTools/uniter-jquery/blob/master/php/src/CountCurrentElements/app.php">Count Current Elements</a>
+                for a look at the php code.
+            </p>
+        </div>
+
+        <div class="full_row">
+            <div class="all_demos" id="demo_element_list">
+                <div class="demo_entry demo_title_entry">
+                    <h2 id="counter">2</h2>
+                </div>
+                <div class="demo_entry demo_title_entry">
+                    <div class="left_cell" id="add_element">
+                        <h2>Add Element</h2>
+                    </div>
+                    <div class="right_cell" id="remove_element">
+                        <h2>Remove Element</h2>
+                    </div>
+                </div>
+                <div class="demo_entry demo_standard_entry">
+                    A demo entry
+                </div>
+                <div class="demo_entry demo_standard_entry">
+                    A demo entry
+                </div>
+            </div>
+        </div>
+
+        <div class="full_row">
+            <h2><a href="/index.html">Back To Home</a></h2>
```

Comment:
Paths need to be relative, eg. just `./index.html` so that they will work with GitHub pages etc. please